samples/apps/bookinfo/consul-content-route.yaml

```diff
@@ -0,0 +1,16 @@
+type: route-rule
+name: content-route
+spec:
+  destination:
+    name: reviews
+    domain: consul
+    namespace: service
+  match:
+    request:
+      headers:
+        cookie:
+          regex: ^(.*?;)?(user=jason)(;.*)?$
+  precedence: 2
+  route:
+    - labels:
+        version: v2
```

Comment:
ideally configure your text editor to automatically add missing last newline

Reply:
thanks, resolved.